docs/lvmd-protocol.md

```diff
@@ -205,6 +205,7 @@ Represents the details of thinpool.
 | data_percent | [double](#double) |  | Data percent occupied on the thinpool, used for monitoring. |
 | metadata_percent | [double](#double) |  | Metadata percent occupied on the thinpool, used for monitoring. |
 | overprovision_bytes | [uint64](#uint64) |  | Free space on the thinpool with overprovision, used for annotating node. |
+| size_bytes | [uint64](#uint64) |  | Size of the thinpool. |
```

Comment:
According to man page. `Physical data space size of the thinpool.` looks explicitly. Could you please consider this?
https://man7.org/linux/man-pages/man7/lvmthin.7.html

Reply:
sounds reasonable, done.